pandas/tests/reshape/test_pivot.py

```diff
@@ -198,6 +210,18 @@ def test_pivot_with_non_observable_dropna(self, dropna):
 
         tm.assert_frame_equal(result, expected)
 
+    def test_pivot_with_interval_index(self, interval_values, dropna):
+        # GH 25814
+        df = pd.DataFrame(
+            {'A': interval_values,
+             'B': [1] * interval_values.size})
+        result = df.pivot_table(index='A', values='B', dropna=dropna)
+        expected = pd.DataFrame(
+            {'B': [1]},
+            index=pd.Index(interval_values.unique(),
```

Comment:
Note that `DataFrame` and `Index` are both already imported, so you can omit the `pd.`